scripts/run-monitor-ci-tests.bash

```diff
@@ -94,8 +94,18 @@ if [[ -z "${OSS_SHA:-}" ]]; then
 		# UI_BRANCH, DEPLOY_PROD are both neccesary to ensure that UI_BRANCH parameter's default in the monitor-ci pipeline doesn't deploy us to production unless started by this script.
 		DEPLOY_PROD=true
 	fi
+	if [ -n $PULL_REQUEST ]; then
+		# if running from pull request, use github api to get source org and branch.
+		# this is needed to support forks of the UI repo.
+		pr=$(echo ${PULL_REQUEST} | grep -oE "[^/]+$")
+		github=$(curl -s --fail --request GET \
+			--url https://api.github.com/repos/influxdata/ui/pulls/${pr})
+		UI_BRANCH=$(echo ${github} | jq -r '.head.ref')
+		UI_GITHUB_ORG=$(echo ${github} | jq -r '.head.repo.owner.login')
+	fi
+
 	pipelineStartMsg="starting monitor-ci pipeline targeting monitor-ci branch ${MONITOR_CI_BRANCH}, UI branch ${UI_BRANCH} and using UI SHA ${SHA}"
-	reqData="{\"branch\":\"${MONITOR_CI_BRANCH}\", \"parameters\":{ \"ui-sha\":\"${SHA}\", \"ui-branch\":\"${UI_BRANCH}\", \"ui-pull-request\":\"${PULL_REQUEST}\", \"deploy-prod\":${DEPLOY_PROD}}}"
+	reqData="{\"branch\":\"${MONITOR_CI_BRANCH}\", \"parameters\":{ \"ui-sha\":\"${SHA}\", \"ui-github-org\":\"${UI_GITHUB_ORG:-influxdata}\", \"ui-branch\":\"${UI_BRANCH}\", \"ui-pull-request\":\"${PULL_REQUEST}\", \"deploy-prod\":${DEPLOY_PROD}}}"
```

Comment:
I have no idea how strict this is, but if we have an org called `sinfluxdata` or `Influxdata` would this match?

Reply:
this is the github org! So in the url `github.com/influxdata/ui`, `influxdata` is the org. For `github.com/genehynson/ui`, `genehynson` is the org

Reply:
Oh are you saying that if someone opened a github account as `sinfluxdata` and tried to open a PR to the UI repo from a fork, how would this work? In that case this is very strict and will attempt to clone a repo at whatever the value actually is. And `Influxdata` with a capital `I` would still clone the non-forked UI repo. Github won't let someone open a new account with `inFluxdAta` because `influxdata` is already taken - the casing doesn't matter

Reply:
Awesome